src/modules/dmCommands/commands/settings.js

```diff
@@ -0,0 +1,103 @@
+import { MessageButton } from 'discord.js';
+import EmbedPage from '../../../dto/EmbedPage';
+import { getMember, getRole, userDb } from '../../../util';
+import EmbedPageMessage from '../../../dto/EmbedPageMessage';
+
+export const command = 'settings';
+export const desc = 'Configure your profile';
```

Comment:
Needs punctuation to be constant, Also your profile, is kinda vague, Maybe more like 
```suggestion
export const desc = 'Configure your user settings.';
```

Reply:
You changed the message here but didn't add puncuation